inorbit_republisher/scripts/republisher.py

```diff
@@ -118,7 +118,9 @@ def callback(msg, repub=repub):
                 elif mapping_type == MAPPING_TYPE_JSON_OF_FIELDS:
                     try:
                         val = extract_values_as_dict(msg, mapping)
-                        val = json.dumps(val)
+                        # We prevent sending a data:"key=null" to the topic
```

Comment:
does it really send `null`? or `None`? in any case, I would not phrase the comment on that side effect you want to patch, but on the reason of doing the filter: We want `extract_values_as_dict` to have the ability to filter out some messages, and returning `None` is the way it signals an element did not pass the filter

Reply:
ah, json.dumps returns `"null"` (the string).. I did not know that. In any case, I would insist on the message focusing on the filter operation, which is the intention of this MR and the feature being implemented

Reply:
yep, I changed the comment too